pkg/koordlet/metricsadvisor/collectors/beresource/be_resource_collector.go

```diff
@@ -58,15 +107,15 @@ func (c *collector) collectBECPUResourceMetric() {
 	}
 
 	collectTime := time.Now()
-	err = c.metricCache.InsertBECPUResourceMetric(collectTime, &beCPUMetric)
+	err = b.metricDB.InsertBECPUResourceMetric(collectTime, &beCPUMetric)
```

Comment:
should we change the metric cache interface while adding a new type of metric?

Reply:
currently, yes it is. because the metric cache is implemented by sqlite, which is a structured DB.
refactor it to TSDB can solve this

Reply:
ok